policyengine_us/variables/gov/states/oh/tax/income/credits/oh_child_dependent_care_credits.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class oh_child_dependent_care_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio senior citizen credits"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=20",
+    )
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        agi_upper_limit = parameters(
```

Comment:
Then access `p.x` in the `{oh,us}_eligible` statements below, directly. For this and other parameters.
```suggestion
        p =   parameters(period).gov.states.oh.tax.income.credits.child
```